src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -40,7 +41,7 @@ trait ClusterComponent extends LeoComponent {
     def status =                column[String]            ("status",                O.Length(254))
     def hostIp =                column[Option[String]]    ("hostIp",                O.Length(254))
     def createdDate =           column[Timestamp]         ("createdDate",           O.SqlType("TIMESTAMP(6)"))
-    def destroyedDate =         column[Option[Timestamp]] ("destroyedDate",         O.SqlType("TIMESTAMP(6)"))
+    def destroyedDate =         column[Timestamp]         ("destroyedDate",         O.SqlType("TIMESTAMP(6)"))
```

Comment:
So is `destroyedDate` becoming not-null? Is there a migration for this?

Also what to do about _existing_ clusters in the DB who have a null `destroyedDate`?

Reply:
I'm willing to sweep this under the rug and deal with it by dropping the dev db for now -- but I agree that this isn't sustainable and we should be migrating in our changesets too.

Reply:
Added the changeset for this.